Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -576,7 +576,20 @@ namespace Multiplayer
                     remoteAddress.c_str()) return;
             }
 
-            AZ_Printf("MultiplayerEditor", "Editor is listening for the editor-server...")
+            // Terminate any existing server launchers before launching a new one.
+            // It's possible for a rogue server launcher to exist if the Editor shutdown unexpectedly while running a previous multiplayer session.
+            // It's also common to open ServerLaunchers by hand for testing, but then to forget to shut it down before starting the editor play mode.
+            const AZStd::string serverExeFilename(AZ::Utils::GetProjectName() + ".ServerLauncher" + AZ_TRAIT_OS_EXECUTABLE_EXTENSION);
+            if (AzFramework::ProcessUtils::TerminateProcess(serverExeFilename))
+            {
+                AZ_Warning("MultiplayerEditorSystemComponent", false,
+                    "Terminating an existing server launcher (%s) before opening a new server. "
+                    "If your intention was to connect to this server instead of automatically launching one from the Editor set editorsv_launch = false.",
+                    serverExeFilename.c_str());
+            }
+
+            AZ_Printf("MultiplayerEditor", "Editor is listening for the editor-server...");
```

Comment:
`AZ_Printf` does not automatically add a newline to the end of its output.
So this call needs to add a newline.
```suggestion
            AZ_Printf("MultiplayerEditor", "Editor is listening for the editor-server...\n");
```

Reply:
But why does it newline in the log?
![image](https://user-images.githubusercontent.com/32776221/221793613-a83b77be-f460-4e21-9256-0877cf904031.png)

Reply:
Because the log file goes through a different path code path where it adds a [missing newline](https://github.com/o3de/o3de/blob/development/Code/Legacy/CrySystem/Log.cpp#L1323-L1347), than when logged to the [CXConsole](https://github.com/o3de/o3de/blob/development/Code/Legacy/CrySystem/Log.cpp#L1358-L1373)